src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto

```diff
@@ -150,6 +150,9 @@ message ImpExt {
     // Supported viewability vendors. For list of supported vendors, visit:
     // https://resources.rubiconproject.com/resource/publisher-resources/xapi-specifications/#4.9.
     repeated string viewabilityvendors = 3;
+
+    // Field for DFP ad unit code and GPID propagation.
```

Comment:
DFP ad unit code is treated in a special way for Magnite only. I would leave a simple comment: "Ad unit code"

Reply:
Thanks, fixed